bin/pycbc_inspiral

```diff
@@ -307,13 +308,16 @@ with ctx:
 
     tsetup = time.time() - tstart
 
+    logging.info("Performing q-transform on analysis segments")
+    out_vals['qtransform'] = qtransform.inspiral_qtransform_generator(segments)
```

Comment:
Does this actually work? If so, interesting, that's not intended but not terrible I guess.